target_chains/solana/programs/solana-receiver/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "solana-receiver"
```

Comment:
I think Pyth should be in the name. Changing the name later is hard.

Reply:
Reasonable. Fixed.